### ISSUE TITLE
I have new amd 6900xt, can i use AMGX to solve my problem. 

### ISSUE THREAD

**Issue by ztdepztdep** [NVIDIA/AMGX#190]
Does AMGX support AMD gpu with opencl.

**Comment by marsaev** [comment]
No, it's CUDA only at the moment. You might try https://github.com/ROCm-Developer-Tools/HIPIFY , but i haven't tried it myself on AMGX.
